app/src/main/java/ch/epfl/qedit/backend/location/LocationService.java

```diff
@@ -0,0 +1,26 @@
+package ch.epfl.qedit.backend.location;
+
+import android.location.LocationListener;
+
+/**
+ * This interface is used to request location updates from the phone. It is an interface, so that it
+ * can be mocked for the tests.
+ */
+public interface LocationService {
+    /**
```

Comment:
Why is it in `backend` package ?

Reply:
Because it is not part of the UI, nor the model. It is an interface that talks with services external to the application, so it feels pretty natural to put it there.